allennlp/predictors/predictor.py

```diff
@@ -103,6 +249,18 @@ def _json_to_instance(self, json_dict: JsonDict) -> Instance:
         """
         raise NotImplementedError
 
+    def predictions_to_labels(self, instance: Instance, outputs: Dict[str, np.ndarray]) -> List[Instance]:
```

Comment:
This needs a better name.  I picked this name originally before we decided that the return value should just be a list of `Instances`.  Maybe `predictions_to_labeled_instances`?